Emitron/Emitron/Data/ViewModels/VideoPlaybackViewModel.swift

```diff
@@ -150,7 +150,7 @@ final class VideoPlaybackViewModel {
     }
     // If we're online then, that's all good
     if sessionController.sessionState == .online {
```

Comment:
If we're commenting out this return we should just remove the whole if block

Reply:
Hi Tim,

I went added that code back in. As for the SD vs HD quality options, I have those in place. For SD streams, I'm downloading the streams at a lower bitrate (250,000). The SD file sizes ended up being half the size as the HD files.